home/mixins.py

```diff
@@ -61,10 +62,25 @@ def _get_stream_data_image_urls(self, raw_data):
 
         return image_urls
 
+    def _get_stream_data_media_urls(self, stream_data):
+        media_urls = []
+
+        for block in stream_data:
+            if block['type'] == 'media':
+                media = Media.objects.filter(id=block['value']).first()
+                if media:
+                    media_urls.append(media.url)
+
+        return media_urls
```

Comment:
This could be replaced with:

```
def collect_media_urls_from_streamfield(field):
    return [ block.value.url for block in field if block.block_type == 'media' ]
```

Where `field` is a StreamField.